templates/buy-me-a-coffee-app/src/pages/index.tsx

```diff
@@ -1,104 +1,52 @@
-import Image from 'next/image';
-import { Inter } from 'next/font/google';
-import { ConnectButton } from '@rainbow-me/rainbowkit';
-
-const inter = Inter({ subsets: ['latin'] });
+import { useCallback } from 'react';
+import { Theme, Box, Container, Flex, Grid, Section, Text } from '@radix-ui/themes';
+import { ThemesHeader } from '@/components/ThemesHeader';
+import { TitleAndMetaTags } from '@/components/TitleAndMetaTags';
+import Head from 'next/head';
+import { FormBuyCoffee, Memos, useBuyMeACoffeeMemos } from '@/features/BuyMeACoffee';
```

Comment:
Can we have some linting rule around import ordering, this feels a bit all over the place.